security/security_profiles_operator/spo-enabling.adoc

```diff
@@ -13,6 +13,11 @@ Before you can use the Security Profiles Operator, you must ensure the Operator
 The Security Profiles Operator supports only Red Hat Enterprise Linux CoreOS (RHCOS) worker nodes. Red Hat Enterprise Linux (RHEL) nodes are not supported.
 ====
 
+[IMPORTANT]
+====
+The Security Profiles Operator only supports `x86_64` architectures.
```

Comment:
Not quite sure if "architecture" is better.
"The Security Profiles Operator only supports `x86_64` architecture"

Reply:
Good point - I'll fix it. Thanks!